{"uuid": "23694928-3d4e-4cc6-988d-f5f0326d1a46", "children": ["9b241081-931b-42b6-9e7d-a1359c01f35a", "b090621c-9cd3-409e-9e79-9e6cf068657d", "36c6938f-cb4d-4321-b989-43892911fb7c"], "befores": [{"name": "host", "status": "broken", "statusDetails": {"message": "configparser.NoOptionError: No option 'url' in section: 'SIT'\n", "trace": "  File \"D:\\Program Files\\python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\Program Files\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\Program Files\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 917, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"F:\\git\\myclone\\AITEST\\conftest.py\", line 25, in host\n    test_host = Conf(env_conf_path).get_value(env, \"url\")\n  File \"F:\\git\\myclone\\AITEST\\common\\conf.py\", line 10, in get_value\n    value=self.cf.get(section,option)\n  File \"D:\\Program Files\\python\\lib\\configparser.py\", line 792, in get\n    raise NoOptionError(option, section)\n"}, "start": 1609825284407, "stop": 1609825284410}], "start": 1609825284407, "stop": 1609825285681}